docs/src/gs/create-workload-cluster.md

```diff
@@ -98,6 +98,55 @@ clusterctl generate cluster <cluster-name>\
 --from cluster-template-oraclelinux.yaml | kubectl apply -f -
 ```
 
+## Create a workload cluster in an alternative region
+
+CAPOCI provides a way to launch and manage your workload cluster in multiple
+regions. Choose the `cluster-template-alternative-region.yaml` template when
+creating your workload clusters from the [latest released artifacts][latest-release].
+Currently, the other templates do not support the ability to change the workload
+cluster region. 
+
+Each cluster can be further configured with the parameters
+defined in [Workload Cluster Parameters](#workload-cluster-parameters) and
+additionally with the parameter below.
+
+| Parameter             | Default Value                                                                | Description                                                                                                                 |
+|-----------------------|------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------|
+| `OCI_WORKLOAD_REGION` | `OCI_REGION` value from [Configure authentication][configure-authentication] | The [region](https://docs.oracle.com/en-us/iaas/Content/General/Concepts/regions.htm) to launch the workload cluster in.    |
+
+Below is an example of configuring the CAPOCI provider to authenticate in
+`us-phoenix-1` then launching the workload cluster in `us-sanjose-1`.
+
+**NOTE** make sure that the region you choose has your image available in that
```

Comment:
Use `>` to blockquote the note. There are also alert templates, I believe. Further, change this to active voice: "Note: ensure the specified image is available in your chosen region or the launch will fail."

Reply:
Great call. I'm going to do this for the rest of the "NOTE" call outs in this file. It really makes it stand out
<img width="768" alt="image" src="https://user-images.githubusercontent.com/132980/162091897-2bf06275-b5ea-4a19-9311-847ffb6779ae.png">